### PR TITLE
BOS-111 fix

### PR DESCRIPTION
**Pull request by pbattu123**
BOS-111 fix

### CODE REVIEW

File: bos_mint/config-defaults.yaml [hunk 1]
```diff
@@ -3,15 +3,16 @@ debug: False
 project_name: MINT
 project_sub_name: The BOS Manual Intervention Module
 secret_key: # enter any random string 
+advanced_features: True
```
Comment (sschiessl-bcp) [inline bos_mint/config-defaults.yaml]: this should not appear in default config

File: bos_mint/config-defaults.yaml [hunk 2]
```diff
@@ -3,15 +3,16 @@ debug: False
...
 
 sql_database: "sqlite:///{cwd}/bookied-local.db"
 
 connection:
-    use: # enter your desired chain
+    use: elizabeth # enter your desired chain
```
Comment (sschiessl-bcp) [inline bos_mint/config-defaults.yaml]: this is default config and should nto work without production config, please leave values as before

File: bos_mint/config-defaults.yaml [hunk 3]
```diff
@@ -3,15 +3,16 @@ debug: False
...
 
-    baxter:
+    elizabeth:
         node:
-            - # enter your node
+            - ws://ec2-18-195-242-233.eu-central-1.compute.amazonaws.com:8090
```
Comment (sschiessl-bcp) [inline bos_mint/config-defaults.yaml]: leave comment as is

File: bos_mint/config-defaults.yaml (hunk 1 above)
Comment (pbattu123) [reply bos_mint/config-defaults.yaml]: fixed

**Pull request merged**